replication/replication_test.go

```diff
@@ -369,6 +369,12 @@ func (t *testSyncerSuite) TestMariadbGTIDSync(c *C) {
 	t.testSync(c, s)
 }
 
+func (t *testSyncerSuite) TestMariadbAnnotateRows(c *C) {
```

Comment:
😎 now we have testing case, but I think it's cooler to confirm that we can receive `annotate rows event`  if enabling in test casing.  What you do think about it @imjustfly

Reply:
Current testing logic is not scalable enough, all testing cases are in `testSync`. And it just ensure there are no errors in event parsing and doesn't check the event type and content. 

If you want to do so, I think it's better to refactor the testing logic and added this event checking at that time.

Reply:
would you like to do it？

Reply:
To be honest, not now. 

I am busy working on syncing data between mariadb now. I just want to fix this issue quickly so I don't need to maintain a `go-mysql` repo by myself. And I think this PR is acceptable at the current existing testing level.  

Refactoring testing should be done by anyone who cares about it, including me, in the future.

Reply:
LOL, feel free